tools/lib/filereader.py

```diff
@@ -12,7 +12,7 @@ def internal_source_available(url=DATA_ENDPOINT):
     hostname = urlparse(url).hostname
     port = urlparse(url).port or 80
     with socket.socket(socket.AF_INET,socket.SOCK_STREAM) as s:
-      s.settimeout(1)
+      s.settimeout(0.5)
```

Comment:
we can also do some heuristic like "username is batman"

Reply:
That one would conflict with the openpilot-base Docker image also using batman as the username

Reply:
let's just make it not batman in the docker?

Reply:
We'd need to make sure any consumers always use the URL

Reply:
I think that for now the timeout is ok. Will merge as is and come back latter to shave off the last seconds to get under 1 minutes consistently (maybe bz2 => zst conversion alone will suffice)